packages/design/src/Colors.mdx

```diff
@@ -24,4 +24,135 @@ ensure you are using colors for their intended purpose.
   alongside color to ensure your content can be understood by everyone.
 </Banner>
 
+## Usage Guidelines
```

Comment:
Ideally there's a nice way to bring the swatches for a section together with the usage guidelines, but until then I think this works. There's something poetic about semantic guidelines being detached from the visuals they describe anyways, right? 😉

Reply:
Using the h2 as an anchor point makes this reasonable to jump between, for now.

![image](https://user-images.githubusercontent.com/39704901/116482675-dc2a4e00-a842-11eb-8515-dc8f95026a75.png)